README.md

```diff
@@ -118,6 +118,45 @@ function ExampleWrapper({
 
 [Try it on Code Sandbox](https://codesandbox.io/s/x70ly749rq)
 
+## Advanced usage
+
+Some use cases require cached items to be reset. For example, after a list has been sorted, previously cached items may be invalid. You can let `InfiniteLoader` know that it needs to reload cached items by calling the `resetloadMoreItemsCache` method.
+
+```jsx
+function ExampleWrapper({
+  // ...
+  sortOrder,
+}) {
+  // We create a reference for the InfiniteLoader
+  const listRef = useRef(null);
+  
+  useEffect(() => {
+    if (listRef.current) {
+      // Each time the sort prop changed we called the method resetloadMoreItemsCache to clear the cache
+      listRef.current.resetloadMoreItemsCache()
```

Comment:
I should have read this more carefully the first time, sorry. This will call `resetloadMoreItemsCache` once unnecessarily after initial mount 😬

Maybe we could use another ref to track this?

```js
const listRef = useRef(null);
const hasMountedRef = useRef(false);

useEffect(() => {
  if (listRef.current && hasMountedRef.current) {
    // Each time the sort prop changed we called the method resetloadMoreItemsCache to clear the cache
    listRef.current.resetloadMoreItemsCache();
  }
  hasMountedRef.current = true;
}, [sortOrder]);
```

Reply:
I changed it (also in the codesandbox), thank you for the review 👍